tests/pyright_test.py

```diff
@@ -16,26 +14,12 @@ def main() -> None:
         print("pyright_test.py must be run from the typeshed root directory", file=sys.stderr)
         sys.exit(1)
 
-    # subprocess.run on Windows does not look in PATH.
-    npx = shutil.which("npx")
-
-    if npx is None:
-        print("error finding npx; is Node.js installed?", file=sys.stderr)
-        sys.exit(1)
-
-    try:
-        subprocess.run([npx, "--version"])
-    except OSError:
-        print("error running npx; is Node.js installed?", file=sys.stderr)
-        sys.exit(1)
-
-    with open("pyproject.toml", "rb") as config:
-        pyright_version: str = tomli.load(config)["tool"]["typeshed"]["pyright_version"]
-
-    os.environ["PYRIGHT_PYTHON_FORCE_VERSION"] = pyright_version
```

Comment:
`os.environ["PYRIGHT_PYTHON_FORCE_VERSION"] = pyright_version` was used in case someone had installed pyright-python in their environment or globally. To ensure that if they set that variable globally on their machine, the command would still follow typeshed's expected version. 

Now that it's listed in our requirements, and we expect des to work with a venv, we could forcibly remove that entry from the environment. Or save the result from the `pyright --version` command and set it here.